<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoctorView</class>
 <widget class="QMainWindow" name="DoctorView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>721</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="crissCross">
     <item>
      <layout class="QVBoxLayout" name="slideToTheLeft">
       <item>
        <widget class="QPushButton" name="addMeds">
         <property name="text">
          <string>Add Medicine</string>
         </property>
         <property name="icon">
          <iconset resource="img.qrc">
           <normaloff>:/img/img/plus.png</normaloff>:/img/img/plus.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addPatient">
         <property name="text">
          <string>Add Patient</string>
         </property>
         <property name="icon">
          <iconset resource="img.qrc">
           <normaloff>:/img/img/new.png</normaloff>:/img/img/new.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="viewPatient">
         <property name="text">
          <string>View Patient</string>
         </property>
         <property name="icon">
          <iconset resource="img.qrc">
           <normaloff>:/img/img/username.png</normaloff>:/img/img/username.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="companyLogo">
       <property name="text">
        <string>Company Logo</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="slideToTheRight">
       <item>
        <widget class="QPushButton" name="Logout">
         <property name="text">
          <string>Logout</string>
         </property>
         <property name="icon">
          <iconset resource="img.qrc">
           <normaloff>:/img/img/logout.png</normaloff>:/img/img/logout.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="seeSchedule">
         <property name="text">
          <string>Nurse Schedule</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reportsButton">
         <property name="text">
          <string>Reports</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
